ush/get_crontab_contents.py

```diff
@@ -36,7 +36,7 @@ def get_crontab_contents(called_from_cron, machine, debug):
     # call the system version of crontab at /usr/bin/crontab.
     #
     crontab_cmd = "crontab"
-    if machine == "CHEYENNE" or machine == "DERECHO":
+    if machine == "CHEYENNE":
         if called_from_cron:
```

Comment:
Is there no Derecho support for the App at the moment?

Reply:
The App currently runs on Derecho.  Given the decommissioning of Cheyenne, I think that `DERECHO` should be kept, while `CHEYENNE` should be removed here.

Reply:
Derecho does not suffer from the same issue that this logic accounts for, see my PR description for why this fix was included (and on a related note, #998).  Cheyenne functionality should be removed in its entirety in a separate PR, I feel that's beyond the scope of this PR.